solutions/swb-reference/src/proxyHandlerLambda.ts

```diff
@@ -48,32 +49,52 @@ export async function handler(event: any) {
   if (event.queryStringParameters) params = event.queryStringParameters;
   if (event.httpMethod) HTTP_METHOD = event.httpMethod;
 
-  if (HTTP_METHOD === 'GET') {
-    const { data, status, statusText } = await axios.get(targetPath, { headers, params });
-    response.statusCode = status;
-    response.statusDescription = statusText;
-    response.body = JSON.stringify(data);
-  }
+  try {
+    if (HTTP_METHOD === 'GET') {
+      const { data, status, statusText } = await axios.get(targetPath, { headers, params });
+      response.statusCode = status;
+      response.statusDescription = statusText;
+      response.body = JSON.stringify(data);
+    }
 
-  if (HTTP_METHOD === 'POST') {
-    const { data, status, statusText } = await axios.post(targetPath, { headers, params, body });
-    response.statusCode = status;
-    response.statusDescription = statusText;
-    response.body = JSON.stringify(data);
-  }
+    if (HTTP_METHOD === 'POST') {
+      const { data, status, statusText } = await axios.post(targetPath, body, { headers, params });
+      response.statusCode = status;
+      response.statusDescription = statusText;
+      response.body = JSON.stringify(data);
+    }
 
-  if (HTTP_METHOD === 'PUT') {
-    const { data, status, statusText } = await axios.put(targetPath, { headers, params, body });
-    response.statusCode = status;
-    response.statusDescription = statusText;
-    response.body = JSON.stringify(data);
-  }
+    if (HTTP_METHOD === 'PUT') {
+      const { data, status, statusText } = await axios.put(targetPath, body, { headers, params });
+      response.statusCode = status;
+      response.statusDescription = statusText;
+      response.body = JSON.stringify(data);
+    }
+
+    if (HTTP_METHOD === 'DELETE') {
+      const { data, status, statusText } = await axios.delete(targetPath, { headers, params });
+      response.statusCode = status;
+      response.statusDescription = statusText;
+      response.body = JSON.stringify(data);
+    }
+  } catch (err) {
+    const error = new HttpError(
```

Comment:
We should de type check. If there is a network issue between proxy and api we won't get response